src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py

```diff
@@ -352,3 +352,24 @@ def test_aks_disable_addons_confcom_addon(self, resource_group, resource_group_l
             self.check('addonProfiles.accsgxdeviceplugin.enabled', False),
             self.check('addonProfiles.accsgxdeviceplugin.config', None)
         ])
+
+    @live_only()
```

Comment:
Is there a special reason to make it `live_only` ?

Reply:
It needs an active cluster to start and stop